solvers/CSPlda.py

```diff
@@ -5,33 +5,61 @@
 # - getting requirements info when all dependencies are not installed.
 with safe_import_context() as import_ctx:
     from sklearn.discriminant_analysis import LinearDiscriminantAnalysis as LDA
-    from mne.decoding import CSP
     from sklearn.pipeline import make_pipeline
-    from benchopt.stopping_criterion import SingleRunCriterion
+    from mne.decoding import CSP
+    from skorch.helper import to_numpy
+
+    from benchmark_utils.transformation import (
+        channels_dropout,
+        smooth_timemask,
+    )
 
 
 # The benchmark solvers must be named `Solver` and
 # inherit from `BaseSolver` for `benchopt` to work properly.
 
 
 class Solver(BaseSolver):
+    name = "CSPLDA"
+    parameters = {
+        "augmentation": [
+            ("SmoothTimeMask"),
+            ("ChannelsDropout"),
+            ("IdentityTransform"),
+        ]
+    }
 
-    name = 'CSPLDA'
+    install_cmd = "conda"
+    requirements = ["mne", "pip:torch", "pip:braindecode"]
```

Comment:
isn't torch a dependency of `braindecode`?

Reply:
also, it feels weird to need `pytorch ` for the baseline model. wouldn't it be possible to avoid this?

Reply:
Yes! Removed